ebpf/aya-ebpf/src/helpers.rs

```diff
@@ -375,7 +364,7 @@ pub unsafe fn bpf_probe_read_user_str(src: *const u8, dest: &mut [u8]) -> Result
 /// [core::str::from_utf8_unchecked]:
 ///
 /// ```no_run
-/// # #![expect(dead_code)]
+/// # #![expect(static_mut_refs)]
```

Comment:
This isn't the fix. Maps haven't needed to be `static mut` for a long time.
The map in the examples can be `static`

Reply:
Thanks for pointing that out! Done.